challenges/objects-arrays.js

```diff
@@ -6,29 +6,68 @@
   object name, diet, weight, length, period
 */
 
+class Dinosaurs {
```

Comment:
nice job making the Dinosaurs class!  I don't think this was required so good job!